docs/topics/mta-cli-transform.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="mta-cli-transform_{context}"]
+= Performing transformation using the command line
+
+Transform has two sub commands - `openrewrite` and `rules`.
+
+[source,terminal,subs="attributes+"]
+----
+Transform application source code or windup XML rules
```

Comment:
@sbeskin-redhat - please can i ask for your advice about this
Should we highlight they have included `windup XML` rules or do you think it is a historical reference and not worth the effort?
Thanks

Reply:
@anarnold97 
Please see my message in Slack.